util/dvsim/BuildOnlyCfg.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""
+Class describing simulation configuration object
+"""
+
+import logging as log
+import sys
+
+from .Deploy import *
+from .FlowCfg import FlowCfg
+from .Modes import *
+from .utils import *
+
+
+class BuildOnlyCfg(FlowCfg):
```

Comment:
"BuildOnly" might make it a little confusing, considering, there is also a `--build-only`switch. How about "OneShotCfg", considering these flows are run in one shot?

Reply:
done